spec.bs

```diff
@@ -543,6 +556,260 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 
 </div>
 
+<div algorithm="construct a pending fenced frame config">
+
+To <dfn>construct a pending fenced frame config</dfn> given an [=auction config=]
+<var ignore>config</var>:

```

Comment:
Can we get rid of this parameter?

Reply:
We will need it later (for the container size).

Reply:
I see. In that case let's reference this variable in the TODO down there just so we can get rid of `ignore`, or just remove it and add it back when we actually use it?

Reply:
Sure, I can reference it in the TODO.